opentelemetry-stdout/examples/basic.rs

```diff
@@ -43,11 +44,11 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     );
     span.end();
 
-    let meter = meter_provider.meter("stdout-test");
+    let meter = global::meter("stdout-test");
```

Comment:
💯 love this! A lot of people asked about how to pass around the provider everywhere, but this neatly solves that problem!